src/Persistence.LinnApps/Repositories/BomVerificationHistoryRepository.cs

```diff
@@ -0,0 +1,66 @@
+﻿namespace Linn.Purchasing.Persistence.LinnApps.Repositories
+{
+    using System;
+    using System.Linq;
+    using System.Linq.Expressions;
+    using Linn.Common.Persistence.EntityFramework;
+    using Linn.Purchasing.Domain.LinnApps.Boms;
+
+    using Microsoft.EntityFrameworkCore;
+
+    public class BomVerificationHistoryRepository : EntityFrameworkRepository<BomVerificationHistory, int>
+    {
+        private readonly ServiceDbContext serviceDbContext;
+       
+        private readonly DbSet<BomVerificationHistory> bomVerificationHistory;
+
+        public BomVerificationHistoryRepository(ServiceDbContext serviceDbContext) : base(serviceDbContext.BomVerificationHistory)
+        {
+            this.serviceDbContext = serviceDbContext;
+        }
+
+        public override BomVerificationHistory FindById(int key)
+        {
+            return this.serviceDbContext.BomVerificationHistory
+                .Include(b => b.PartNumber)
+                .Include(b => b.DateVerified)
+                .Include(b => b.VerifiedBy)
+                .Include(b => b.DocumentType)
```

Comment:
as lewis said before you don't need any of these includes. they're not doing anything